chromium_src/chrome/browser/ui/views/first_run_dialog.cc

```diff
@@ -72,7 +73,7 @@ FirstRunDialog::FirstRunDialog(base::RepeatingClosure learn_more_callback,
     : quit_runloop_(quit_runloop) {
   ALLOW_UNUSED_LOCAL(report_crashes_);
 
-  SetTitle(l10n_util::GetStringUTF16(IDS_FIRST_RUN_DIALOG_WINDOW_TITLE));
+  SetTitle(l10n_util::GetStringUTF16(IDS_FIRSTRUN_DIALOG_WINDOW_TITLE_BRAVE));
```

Comment:
Upstream doesn't include `IDS_FIRST_RUN_DIALOG_WINDOW_TITLE` for Windows.
So, `IDS_FIRSTRUN_DIALOG_WINDOW_TITLE_BRAVE` is defined and used on all platforms.